--- meson.build~	2021-11-24 12:15:58.564374000 -0600
+++ meson.build	2022-01-06 09:43:04.302267338 -0600
@@ -392,11 +392,7 @@
 configure_file(output : 'config.h', configuration : cdata)
 
 subdir('libsoup')
-xgettext = find_program('xgettext', required : false)
-# xgettext is optional (on Windows for instance)
-if xgettext.found()
-  subdir('po')
-endif
+subdir('po')
 
 subdir('examples')
 subdir('fuzzing')
@@ -421,7 +417,7 @@
     'GSSAPI' : enable_gssapi,
     'NTLM' : ntlm_auth.found(),
     'Brotli' : brotlidec_dep.found(),
-    'Translations' : xgettext.found(),
+    'Translations' : 'True',
     'GIR' : enable_introspection,
     'VAPI' : enable_vapi,
     'Documentation' : get_option('gtk_doc'),
